src/wallet/rpcwallet.cpp

```diff
@@ -2450,6 +2450,32 @@ static UniValue getwalletinfo(const JSONRPCRequest& request)
     return obj;
 }
 
+static UniValue listwalletdir(const JSONRPCRequest& request)
```

Comment:
Ditto as above, but unfortunately it's too late to rename `listwallets` to `listloadedwallets`. Overloading `listwallets` doesn't seem like a good idea.

What about `lswallets`, `listwalletfiles` or `findwallets`?